### DISCUSSION TITLE
Is Quarto in rocker images yet?

### DISCUSSION THREAD

**Discussion by jimjam-slam** [quarto-dev/quarto-cli#1985] [Q&A]
This might be a bit out of Quarto's court, but I was wondering whether it had made it into any rocker images yet or was likely to in the future. I know `rocker/verse` is specifically intended to have publishing tools added, but since quarto is bundled with n ewer versions of RStudio and many of the `rocker` images come with RStudio Server added,  I thought it might be the case that Quarto was already hanging around in some of them.

I'm considering setting up devcontainers for our open analyses and charts going forward to assist with reproducibility, so it would be nice if we could base off a rocker image that had Quarto included! (Of course, we can set up an image that adds Quarto; I'm just interested to see if it's already there with RStudio Server!)

**Comment by jimjam-slam** [comment]
Ah, I hadn't caught that [`rocker/verse` already has a Quarto install script bundled!](https://github.com/quarto-dev/quarto-cli/discussions/439#discussioncomment-2491441)  Awesome!

**Reply by mcanouil** [reply]
Only 4.2.1 (maybe 4.2.0) if I recall properly.

**Comment by eitsupi** [comment]
I know this is a late reply, but `rocker/verse` has had quarto-cli installed since last year. rocker-org/rocker-versioned2#253

Such changes are noted in the NEWS file.
https://github.com/rocker-org/rocker-versioned2/blob/master/NEWS.md
